src/99-appendices/08-coordinate-systems.md

```diff
@@ -195,6 +195,11 @@ Unless specified explicitly in the sidecar file in the
 
 ### Standard template identifiers
 
+When using any of the below templates, coordinates should be in "world" (RAS+) space,
+and not voxel indices.
+If `<CoordSysType>CoordinateUnits` is not defined or applicable, then the units SHALL
+be assumed to be millimeters (mm).
```

Comment:
Fair enough. I missed that this was a subheading of **Image-based Coordinate Systems**.

```suggestion
```